### ISSUE TITLE
esp32 Cam implementation

### ISSUE THREAD

**Issue by kleffa** [probonopd/WirelessPrinting#182]
anyone have pinout to conect SD to boards or it can be setup by thae code?
Im trying to flah it to a esp32cam, wich comes with a SD card slot, but not sure if pins are already

**Comment by probonopd** [comment]
Good idea to use the esp32cam due to its built-in SD slot. I don't think I have tried using it so far.

Probably you would have to adjust 

https://github.com/probonopd/WirelessPrinting/blob/1f4a381489fd458cd358db380c1d058e162b4706/ESP8266WirelessPrintAsync/StorageFS.h#L20

to match the esp32cam.

**Comment by kleffa** [comment]
it should be         
SPI.begin(14, 12, 13, 15); // esp32cam built in microSD{
what I'm doing wrong? by now SD is not working... what happend with GPIO 4 and GPIO 2? those should 

by now werbserver and octoprint plugin work, some comunication to printer... but doesnt show ip on screen and webserver didnt show printers name (any idea?)
I tried the last relase of a wemos D1, it show ip and SPIFFS, some kind of work, but when send print from pc, it just say terminated, so I´m not sure if has point???

here some changes I had made to work serial 

in ESP8266Wiresprint...ino  I make this 
#define LED_BUILTIN 33
HardwareSerial PrinterSerial(0); //esp32 cam only have this
PrinterSerial.begin(serialBauds[serialBaudIndex], SERIAL_8N1, 3 ,1 ); // gpio3 = rx, gpio1 = tx

werbserver works and get response from printer, EX
Free heap: 197268

File system: SPIFFS
Filename length limit: 11
Uploaded file: cached.gco
Uploaded file size: 272477

Last command sent: M115
Last received response:  0x400805f0"ok

also set definitions for this things to exchange for TTgoT1 or simple make changes for other esp32 models

**Comment by probonopd** [comment]
Actually now I vaguely remember that I had looked at this some time ago and had concluded that the pinout of th ESP32Cam wasn't compatible. Don't remember the details though, and I might be wrong, so please take with a grain of salt. Anyhow, I don't know how to make it work on this board.

**Comment by kleffa** [comment]
I read back some issues, I think problem was that serial UART0 is at GPIO 1
and 3, but never planted what if setup serial after and set to pin 0 and 16

**Comment by probonopd** [comment]
So do we agree that it is not possible with this hardware?
